modules/mixins/cacheMixin.js

```diff
@@ -148,8 +148,7 @@
 
             //find ids that we don't have in cache and aren't already in the process of being fetched.
             var idsNotInCache = _.difference(this.getTrackedIds(), _.pluck(parentInstance.models, 'id'));
-            var allExistingPromisesForIds = _.pick(parentInstance.idPromises, idsNotInCache);
-            var idsWithPromises = _.mapObject(allExistingPromisesForIds, _.isEmpty);
+            var idsWithPromises = _.pick(parentInstance.idPromises, idsNotInCache);
```

Comment:
removed the case where an id could have an empty array.  _.mapObject(allExistingPromisesForIds, _.isEmpty); didn't do what I expected it to.